cookbook.go

```diff
@@ -38,7 +38,7 @@ type CookbookVersion struct {
 
 // CookbookMeta represents a Golang version of cookbook metadata
 type CookbookMeta struct {
-	Name            string                 `json:"cookbook_name,omitempty"`
+	Name            string                 `json:"name,omitempty"`
```

Comment:
I also fixed the `CookbookMeta` struct, here is an example of the `metadata` of a cookbook coming from a Chef Server:
```
  "metadata": {
    "name": "foo",
    "description": "Installs/Configures foo",
    "long_description": "Installs/Configures foo",
    "maintainer": "The Authors",
    "maintainer_email": "you@example.com",
    "license": "All Rights Reserved",
    "platforms": {},
    "dependencies": {},
    "recommendations": {},
    "suggestions": {},
    "conflicting": {},
    "providing": {
      "foo": ">= 0.0.0"
    },
    "replacing": {},
    "attributes": {},
    "groupings": {},
    "recipes": {
      "foo": ""
    },
    "version": "0.3.0"
  },
```